netket/sampler/jax/metropolis_hastings.py

```diff
@@ -132,7 +132,7 @@ def generate_samples(self, n_samples, init_random=False, samples=None):
         self.reset(init_random)
 
         self._rng_key, samples = self._metropolis_kernel(
-            self.machine.jax_forward_nj,
+            self.machine._forward_fn_nj,
```

Comment:
@PhilipVinc so from now on we just access the non-jitted forward function like this whenever we need it internally?

Reply:
yes.